### PR TITLE
Facebook callback handling

### PR DESCRIPTION
**Pull request by raajgopal94**

### CODE REVIEW

File: app/src/main/java/com/bigmenu/activities/LoginActivity.java [hunk 1]
```diff
@@ -100,6 +113,48 @@ private void custimizeSignBtn() {
...
+                GraphRequest request = GraphRequest.newMeRequest(
+                        loginResult.getAccessToken(),
+                        new GraphRequest.GraphJSONObjectCallback() {
+                            @Override
+                            public void onCompleted(JSONObject object, GraphResponse response) {
+                                Log.v("LoginActivity", response.toString());
+
+                               try {
+                                   String email = object.getString("email");
+                                   String birthday = object.getString("birthday");
```
Comment (raajgopal94) [inline app/src/main/java/com/bigmenu/activities/LoginActivity.java]: Access the required Info from Object here.

Comment (gvnbsg) [inline app/src/main/java/com/bigmenu/activities/LoginActivity.java]: How to get the friends list? Is it also String?

Comment (raajgopal94) [inline app/src/main/java/com/bigmenu/activities/LoginActivity.java]: {"summary":{"total_count":10},"data":[]} list will come as a part of data need to handle that..

File: app/src/main/java/com/bigmenu/activities/LoginActivity.java [hunk 2]
```diff
@@ -100,6 +114,62 @@ private void custimizeSignBtn() {
...
+                                   e.printStackTrace();
+                               }
+                            }
+                        });
+                GraphRequest requestFriends = GraphRequest.newMyFriendsRequest(
+                        loginResult.getAccessToken(),
+                        new GraphRequest.GraphJSONArrayCallback() {
+                            @Override
+                            public void onCompleted(JSONArray object, GraphResponse response) {
+                                Log.v("LoginActivity", response.toString());
```
Comment (raajgopal94) [inline app/src/main/java/com/bigmenu/activities/LoginActivity.java]: JSONArray has {"summary":{"total_count":10},"data":[]} , need to look on how we can access that data

**Pull request merged**